app-config.yaml

```diff
@@ -306,78 +306,78 @@ auth:
   # session:
   #   secret: custom session secret
   providers:
-    google:
-      development:
-        clientId: ${AUTH_GOOGLE_CLIENT_ID}
-        clientSecret: ${AUTH_GOOGLE_CLIENT_SECRET}
+    # google:
```

Comment:
There are a few unrelated changes like this one and a new $env file. Make sure to revert/delete those :)